src/styles/index.scss

```diff
@@ -1,3 +1,143 @@
+$stars_size: 16px;
+
 body {
   margin: 0;
+  font-size: 10px;
+  font-weight: 400;
+  font-family: Roboto, sans-serif;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: #{$stars_size};
+  height: #{$stars_size};
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
```

Comment:
The `margin-left` value of `4px` could be extracted into a variable for consistency and ease of maintenance, especially if it's a value that could be reused in other parts of the style.